src/ia-book-actions.js

```diff
@@ -580,14 +598,6 @@ export default class IABookActions extends LitElement {
     ></book-title-bar>`;
   }
 
-  get timerCountdownTemplate() {
```

Comment:
i moved this to render and used CSS `hide` to remove it from display. this way it is directly tied to `this.lendingStatus` lifecycle updates